realm/realm-library/src/main/java/io/realm/log/RealmLogger.java

```diff
@@ -22,6 +22,7 @@
  * Interface for custom loggers that can be registered at {@link RealmLog#add(RealmLogger)}.
  * The different log levels are described in {@link LogLevel}.
  */
+@KeepMember
```

Comment:
So the `@KeepMember` has to be applied to the class as well? Is it still necessary to apply `@KeepMember` for the `log` method below?